paymentsheet/src/main/java/com/stripe/android/customersheet/data/CustomerSheetDataRepository.kt

```diff
@@ -0,0 +1,87 @@
+package com.stripe.android.customersheet.data
+
+import com.stripe.android.customersheet.ExperimentalCustomerSheetApi
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodUpdateParams
+
+@ExperimentalCustomerSheetApi
+internal interface CustomerSheetDataRepository {
```

Comment:
How is this different than CustomerAdapter?

How would you implement this as customer adapter?

As is, I don't think there's enough information to review how it's going to work. I think we'd need to see an implementation.

Reply:
Added a `CustomerAdapterDataRepository` implementation. This was always intended to be a wrapper around `CustomerAdapter` with extra operations to support loading `ElementsSession`.

One thing I did identity is that there is going to be a DI burden that needs to be maintained. One component for `CustomerAdapterDataRepository` and one for `CustomerSessionDataRepository`. We can mitigate this as much as possible with a `Common` module. What do you think?